src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -2815,16 +2819,16 @@ internal static bool TryGetRelativeEntitySetPath(IEdmElement element, Collection
             }
 
             // If there is no parameter then this will fail in BoundOperationMustHaveParameters rule so skip validating this here.
-            if (!parameters.Any())
+            parameter = parameters.FirstOrDefault();
+            if (parameter == null)
             {
                 return false;
             }
+            Debug.Assert(parameter != null, "Should never be null");

```

Comment:
not sure this assert is necessary, since previous line would have filtered out a null parameter value.

Reply:
I think so. Removed.

---
In reply to: [290008471](https://github.com/OData/odata.net/pull/1473#discussion_r290008471) [](ancestors = 290008471)